src/test/TestSmoothTree.cpp

```diff
@@ -13,6 +13,7 @@
 #include "../utils/UrlUtils.h"
 
 #include <gtest/gtest.h>
+#include <decrypters/Helpers.h>
```

Comment:
and moved above by sorting it
```suggestion
#include "../decrypters/Helpers.h"
```